### DISCUSSION TITLE
stellar主题添加代码块复制功能 - WxW

### DISCUSSION THREAD

**Discussion by giscus** [Whbbit1999/wxw-blog-comments#4] [General]
# stellar主题添加代码块复制功能 - WxW

给 stellar 主题添加代码块复制功能

https://www.whbbit.cn/2023/01/31/stellar%E4%B8%BB%E9%A2%98%E6%B7%BB%E5%8A%A0%E4%BB%A3%E7%A0%81%E5%9D%97%E5%A4%8D%E5%88%B6%E5%8A%9F%E8%83%BD/

**Comment by Achuan-2** [comment]
感谢！
